enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -752,6 +754,12 @@ func (mc *MigrationCache) Writer(ctx context.Context, r *rspb.ResourceName) (int
 		return nil, err
 	}
 
+	if mc.rateLimitWriteToDest {
+		// We will write to the destination cache in the background
+		mc.sendNonBlockingCopy(ctx, r, false /*=onlyCopyMissing*/)
```

Comment:
onlyCopyMissing should be true to avoid enqueing copies for digests that already exist

Reply:
Talked offline. Somewhere in the system checks whether the digest exists in the system or not prior to Write() is called. So we don't need to set onlyCopyMissing = true.